### PR TITLE
Added a global variable to check if service finder is active to added…

### PR DESCRIPTION
**Pull request by Buhle79**
… find clinic and service to the menu

### CODE REVIEW

File: yal/rapidpro.py [hunk 1]
```diff
@@ -161,3 +161,34 @@ async def check_if_baseline_active():
...
+        is_service_finder_active = False
+        for i in range(3):
+            try:
+                response = await session.get(
+                    urljoin(
+                        config.RAPIDPRO_URL,
+                        "/api/v2/globals.json?key=service_finder_active",
+                    ),
+                )
+                print(">>>>>", response)
```
Comment (fritzbrand) [inline yal/rapidpro.py:180]: We can probably remove this print?

**Review by fritzbrand — COMMENTED** [review]
Buhle please remember to add the var to Prod as well, with a False value.

Did you get someone to explain that `rapidpro_mock.tstate.requests` thing to you?

Other than that, it looks good to me

**Comment by Buhle79** [comment]
@fritzbrand 

Can you please grant me access to Prod i can only access QA.

I will ask Erik to explain to me before i can deploy incase we interpreted it wrong.

**Comment by fritzbrand** [comment]
@Buhle79 ok cool, prd access granted.

**Review by erikh360 — APPROVED** [review]

**Pull request merged**